packages/react/src/components/Pagination/usePagination.ts

```diff
@@ -64,13 +64,13 @@ export const usePagination = ({
         (page, index) => ({
           page: page || '',
```

Comment:
```suggestion
          page,
```

Reply:
Can be now that we type check `typeof page === 'number'` ☺️

Reply:
Aye, was thinking let it be up to the end user to decide how they want to check if page is a page or ellipsis, be it typeof or if `ellipsis `